tools/codegen/gen.py

```diff
@@ -219,10 +212,10 @@ def func(f: NativeFunction) -> Optional[str]:
         returns_type = native.returns_type(f.func.returns)
         args = native.arguments(f.func)
         args_str = ', '.join(map(str, args))
-        # TODO: don't need dispatch is None shortly
-        dispatch_to_all_backends = dispatch is None or dispatch in KEYWORD_ALL_BACKENDS
+        dispatch_to_all_backends = dispatch is not None and dispatch in KEYWORD_ALL_BACKENDS
```

Comment:
nit: we don't really need the `is not None` check since KEYWORD_ALL_BACKENDS doesn't contain `None`

Reply:
Actually we do, because `KEYWORD_ALL_BACKENDS` is typed as a dict from string to thing, so a None value woudn't be well typed 👍